transitfeed/util.py

```diff
@@ -226,9 +226,10 @@ def CheckVersion(problems, latest_version=None):
 
 
 def _MaxVersion(versions):
-  versions = filter(None, versions)
-  versions.sort(lambda x,y: -cmp([int(item) for item in x.split('.')],
-                                 [int(item) for item in y.split('.')]))
+  versions = [item for item in versions if item]
+  version_tuple = lambda x: tuple([int(item) for item in x.split('.')])
```

Comment:
No need to materialize the the values to a list and then to a tuple. The square brackets materialize a list. Instead of `tuple([whatever])` do `tuple(whatever)`

Reply:
done.